docs/logbook/LHC/2025_lhc.md

```diff
@@ -31,7 +31,9 @@ The tables below can be sorted by clicking next to the column headers.
 | 2025-05-07 20:00 | 2025-05-08 07:30 |    1.5WN     | Commissioning |                Ion Optics Part 1                 |      [Shift Start](2025-05-07, 1081, 4257152){.logbook-link} / [Summary](2025-05-08, 1081, 4257277){.logbook-link} ([OP](https://logbook.cern.ch/elogbook-server/GET/showEventInLogbook/4258231){.cern_login})      |
 | 2025-05-18 13:00 | 2025-05-18 19:00 |  0.5H 0.25HN | Commissioning |                Ion Optics Part 2                 |      [Shift Start](2025-05-18, 1081, 4295338){.logbook-link} / [Summary](2025-05-18, 1081, 4295555){.logbook-link} ([OP](https://logbook.cern.ch/elogbook-server/GET/showEventInLogbook/4295553){.cern_login})      |
 | 2025-06-13 16:30 | 2025-06-13 20:00 |  0.2W 0.3WN  | Commissioning |   Hierarchy breaking investigation at Injection  | [Shift Start](2025-06-13, 1081, 4315743){.logbook-link} / [Summary](2025-06-13, 1081, 4315800){.logbook-link} ([OP](https://be-op-logbook.web.cern.ch/elogbook-server/GET/showEventInLogbook/4315806){.cern_login}) |
-| 2025-06-16 
+| 2025-06-16 13:20 | 2025-06-16 16:00 |     0.4W     | Commissioning |                Ion Optics Part 3                 |                                               [Shift Start](2025-06-16, 1081, 4317342){.logbook-link}                                                                                                               |
+| 2025-06-21 23:00 | 2025-06-22 07:00 |     1HN      |    MD15463    |       LRBB compensation with flat-optics         | [Shift Start](2025-06-21, 1081, 4321399){.logbook-link} /<br>[OP Summary](https://be-op-logbook.web.cern.ch/elogbook-server/GET/showEventInLogbook/4321626){.cern_login}                                            |
```

Comment:
Let's link to the summary in the OMC logbook (event ID 4324656) and also the OP ones. Should be a Summary (OP) like other entries.